content/docs/integrations/istio/sidecar/ingress.md

```diff
@@ -139,24 +165,35 @@ If you have a revisionless istiod setup and did not customize the Istio meta clu
    EOF
    ```
    
-5. Verify that the gateway proxy is now successfully deployed. You might need to restart the proxy to apply the latest Gateway settings. 
+2. Verify that the gateway proxy is now successfully deployed. You might need to restart the proxy to apply the latest Gateway settings. 
```

Comment:
why might? i feel like we should be able to say that you either need to restart it, or it picks up the settings on its own?

Reply:
I rephrased it to a troubleshooting step incase they don't see the containers...I think it could happen if the Gateway already existed (although it didn't do that for me, but I think Nadine ran into it when she first wrote this topic)